app/controllers/item.js

```diff
@@ -81,6 +83,23 @@ export default Ember.Controller.extend({
       this.set('previewUrl', image.get("previewImageUrl"));
     },
 
+    gotoStockItem(inventoryNumber) {
+      let environment = config.staging ? "staging" : config.environment;
+      let finalUrl;
+
+      switch(environment) {
```

Comment:
Refactor switch case to use **config.APP.BASE_HOST_URL.**
I don't think we need to compare environments here.